webapp/i18n/en.json

```diff
@@ -540,6 +540,24 @@
   "admin.saml.usernameAttrTitle": "Username Attribute:",
   "admin.saml.verifyDescription": "When true, Mattermost verifies that the signature sent from the SAML Response matches the Service Provider Login URL",
   "admin.saml.verifyTitle": "Verify Signature:",
+  "admin.cluster.loadedFrom": "This config file was loaded from Node ID {clusterId}.  If you are accessing the System Console through a load balancer you may experience issues.",
```

Comment:
"config" > "configuration". Should we link to the documentation where we would have a troubleshooting guide?

`This configuration file was loaded from Node ID {clusterId}. Please see the Troubleshooting Guide in our <a href=\"http://docs.mattermost.com/deployment/cluster.html\" target=\"_blank\">documentation</a> if you are experiencing issues accessing the System Console through a load balancer.`